backend/routes/rides.js

```diff
@@ -56,7 +66,9 @@ router.get('/', async function (req, res, next) {
         // })
         .toArray(function (err, rides) {
             if (err) {
-                console.log(err);
```

Comment:
Shouldn't it be better if we always console log these err/request query, so we can quickly determine if there's any problem even after the website is deployed?

Reply:
You're right, I agree. I just wrapped it in a if (DEBUG) statement b/c my console was getting too cluttered. let me change it back.